sql_generators/glean_usage/templates/dau_reporting_clients_first_seen_v1.metadata.yaml

```diff
@@ -0,0 +1,19 @@
+{{ header_yaml }}
+friendly_name: DAU Reporting Clients First Seen
+description: |-
+  A representation of when we saw each `client_id`
+  for the first time based on the dau_reporting ping.
+
+owners:
+  - kik@mozilla.com
+labels:
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_glean_usage
+  task_group: {{ app_name }}
+bigquery:
+  time_partitioning:
+    type: day
+    field: submission_date
+    require_partition_filter: true
```

Comment:
I have long held the opinion that `submission_date` fields for `first_seen` tables don't make sense. The grain is `client`, with a `first_seen` fact. The `submission_date` doesn't really have any relevance for a particular client (and is _especially_ confusing when one has to filter on the field!).

Reply:
I tried to make an adjustment to this specific model so that we only have a first_seen_date date field. I kind of agree with you there.